src/agent/debuglet.js

```diff
@@ -339,7 +347,10 @@ Debuglet.prototype.getProjectId_ = function(callback) {
         if (!project) {
           return callback(err);
         }
-        return callback(null, project, onGCP);
+        metadata.instance(
+          'attributes/cluster-name', function(err, response, metadataCluster) {
+            return callback(null, project, metadataCluster, onGCP);
```

Comment:
Now that we have three return values, it would be better to return an object with keys rather than positional arguments.